components/browser/domains/src/main/java/mozilla/components/domains/Domains.kt

```diff
@@ -0,0 +1,78 @@
+package mozilla.components.domains
+
+import android.content.Context
+import android.os.LocaleList
+import android.text.TextUtils
+import java.io.IOException
+import java.util.Locale
+
+/**
+ * Contains functionality to access domain lists shipped as part of this
+ * module's assets.
+ */
+object Domains {
+
+    /**
+     * Loads the domains applicable to the app's locale, plus the domains
+     * in the 'global' list.
+     *
+     * @param context the application context
+     * @return list of domains
+     */
+    suspend fun load(context: Context): List<String> {
```

Comment:
AS is complaining that 'suspend' is redundant here because there's no call to other suspending functions in here. And after removing it here it complains about other places too.

Reply:
Hm, it does call a suspending function though. It's also used in launch(UI) context so it definitely should be suspendable, no?

https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/java/org/mozilla/focus/autocomplete/UrlAutoCompleteFilter.kt#L97